docs/release-notes.md

```diff
@@ -59,6 +59,9 @@ UNRELEASED
 * Fixed "stb-tester fails with PyGObject 3.13" #305.  This will allow
   stb-tester to work out of the box on Ubuntu >= 14.10 and Fedora >= 21.
 
+* `wait_until` can now take a `functools.partial` object. Thanks to Martyn
+  Jarvis for the patch.
```

Comment:
One nitpick, `wait_until` could already take partials, as long as the condition waited for would occur during the timeout.